packages/flutter_tools/test/general.shard/isolated/windows/native_assets_test.dart

```diff
@@ -145,7 +145,10 @@ void main() {
             'package:bar/bar.dart',
             if (flutterTester)
               // Tests run on host system, so the have the full path on the system.
-              projectUri.resolve('build/native_assets/$expectedDirectory/bar.dll').toFilePath()
+              projectUri
+                  .resolve('build/native_assets/$expectedDirectory/bar.dll')
+                  .toFilePath()
+                  .replaceAll(r'\', r'\\')
```

Comment:
why this replacement (add a comment maybe?).

Reply:
We went from yaml to json, and json adds an extra layer of escaping of backslashes in strings.